.idea/HW-Week2_Day03_Inheritance_and_ClassTypes.iml

```diff
@@ -1,9 +1,68 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
+//file1
+fun main() {
+    val c1= Circle() //creating object from the subclass
+    val r1 = Rectangular() //creating object from the subclass
+    val t1 = Triangle() //creating object from the subclass
+    c1.computeArea() //call the fun from the sub class
+    r1.computeArea() //call the fun from the sub class
+    t1.computeArea() //call the fun from the sub class
+
+
+}
+open class Shape { //create class
+    val shapeName =" " //declaration of variable
+    val shapeColor = " " //declaration of variable
+    open fun info(color:String,shape:String): String { //create fun
+        return color+shape // return value
+
+    }
+    open fun computeArea(){ //create fun
+
+    }
+}
+
+//file2
+class Circle:Shape() { //inherit the super class
+    val pi = 3.14
+    val radius = 0
+
+     override fun computeArea() { //override the fun
+    println("Enter radius")
```

Comment:
why use override ? what dose it change exactly ?what effect dose it have on the rest of the code ?